dev-guide/cluster-version-operator/dev/operators.md

```diff
@@ -165,3 +175,5 @@ Bundle up your manifests and the metadata file as a docker image and push them t
 Later on, when someone wants to mirror a particular release, there will be tooling that can take the list of all images used by operators and mirror them to a new repo.
 
 This pattern tries to balance between having the manifests in your source repo be able to deploy your latest upstream code *and* allowing us to get a full listing of all images used by various operators.
+
+[cluster-profiles]: https://github.com/openshift/enhancements/blob/master/enhancements/update/cluster-profiles.md
```

Comment:
no need for this definition, now that you have a local, inline link above.

Reply:
fixed